docs/README.md

```diff
@@ -0,0 +1,14 @@
+# 💊  ReVanced Manager
+
+The official guide for ReVanced Manager.
+
+For bug reports, suggestions, or feature requests, open an issue [here](https://github.com/revanced/revanced-manager/issues/new/choose). If you wish to discuss the Manager further, a thread has been made under the [#development](https://discord.com/channels/952946952348270622/1002922226443632761) channel in our [Discord server](https://revanced.app/discord).
+
```

Comment:
Other docs, such as that of ReVanced CLI or ReVanced Patches, do not include this text; either consistently add everywhere or remove it here. You can also include it only in the advanced-documentation repo.